### ISSUE TITLE
add a synchronous test client

### ISSUE THREAD

**Issue by eugeniy** [eugeniy/pytest-tornado#5]
fetch() should take a path, use base_url to convert it to an absolute url for the test server.

Needs to work inside the gen_test decorator, so probably can't just run_sync in both places.

**Comment by dingyaguang117** [comment]
Yes, `gen_test` can't work with `mock.patch`, a synchronous client is necessary.
